src/main/java/li/cil/oc2/common/entity/RobotEntity.java

```diff
@@ -332,21 +331,16 @@ public boolean canBeCollidedWith() {
     }
 
     @Override
-    public boolean canCollide(final Entity entity) {
+    public boolean canCollideWith(final Entity entity) {
         return entity != this;
     }
 
     @Override
-    public void applyEntityCollision(final Entity entity) {
+    public void push(final Entity entity) {
     }
 
     @Override
-    public boolean func_241845_aY() {
```

Comment:
Why was this removed? This is what allows players to stand on top of robots.

Reply:
Thanks this fixed the bug where i couldn't interact with the robot